src/features/dotnetTest.ts

```diff
@@ -143,7 +156,12 @@ export function debugDotnetTest(testMethod: string, fileName: string, testFramew
                 serverUtils.debugTestRun(server, { FileName: fileName });
```

Comment:
We'll want to trigger this after `processLaunched` event is received and pass the process ID to OmniSharp. I can update OmniSharp's protocol to take this.